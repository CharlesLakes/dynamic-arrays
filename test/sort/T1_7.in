10
92 92 85 48 67 92 79 58 8 6