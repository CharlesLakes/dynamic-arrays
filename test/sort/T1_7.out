6 8 48 58 67 79 85 92 92 92